docs/en/ingest-management/fleet/fleet-settings-output-kafka.asciidoc

```diff
@@ -0,0 +1,367 @@
+
+* Plain
+* SCRAM-SHA-256
+* SCRAM-SHA-512
+
+// ============================================================================
+
+|
+[id="kafka-output-authentication-ssl"]
+**SSL**
+
+| Authenticate using the Secure Sockets Layer (SSL) protocol. Provide the following details for your SSL certificate:
+
+// Kafka SSL setting descriptions match Logstash SSL settings: https://www.elastic.co/guide/en/fleet/current/fleet-settings.html#ls-output-settings
+
+Client SSL certificate::
+The certificate generated for the client. Copy and paste in the full contents of the certificate. This is the certificate that all the agents will use to connect to Kafka.
++
+In cases where each client has a unique certificate, the local path to that certificate can be placed here. The agents will pick the certificate in that location when establishing a connection to Kafka.
+
+Client SSL certificate key::
+The private key generated for the client. This must be in PKCS 8 key. Copy and paste in the full contents of the certificate key. This is the certificate key that all the agents will use to connect to Kafka.
++
+In cases where each client has a unique certificate key, the local path to that certificate key can be placed here. The agents will pick the certificate key in that location when establishing a connection to Kafka.
+
+// ============================================================================
+
+|
+**Server SSL certificate authorities**
+
+| The CA certificate to use to connect to Kafka. This is the CA used to generate the certificate and key for Kafka. Copy and paste in the full contents for the CA certificate.
+
+This setting is optional.
+
+Click **Add row** to specify additional certificate authories.
+
+// ============================================================================
+
+|
+**Verification mode**
+// from beats/libbeat/docs/shared-ssl-config.asciidoc
+
+| Controls the verification of server certificates. Valid values are:
+
+`Full`::
+Verifies that the provided certificate is signed by a trusted
+authority (CA) and also verifies that the server's hostname (or IP address)
+matches the names identified within the certificate.
+
+`None`::
+Performs _no verification_ of the server's certificate. This
+mode disables many of the security benefits of SSL/TLS and should only be used
+after cautious consideration. It is primarily intended as a temporary
+diagnostic mechanism when attempting to resolve TLS errors; its use in
+production environments is strongly discouraged.
+
+`Strict`::
+Verifies that the provided certificate is signed by a trusted
+authority (CA) and also verifies that the server's hostname (or IP address)
+matches the names identified within the certificate. If the Subject Alternative
+Name is empty, it returns an error.
+
+`Certificate`::
+Verifies that the provided certificate is signed by a
+trusted authority (CA), but does not perform any hostname verification.
+
+The default value is `Full`.
```

Comment:
We might consider reiterating in both these sections that users who select None/Plaintext authentication will not have access to these options. This will address the situation where someone is seeking an explanation for not seeing the CA or Verification field.

Reply:
For `Server SSL certificate authorities` I updated the text to:
> This setting is optional. This setting is not available when the authentication `None` and `Plaintext` options are selected.

And for `Verification mode`: 
> The default value is `Full`. This setting is not available when the authentication `None` and `Plaintext` options are selected.